flow/project.py

```diff
@@ -687,7 +688,7 @@ def __call__(self, func):
     def _set_directives(self, func, directives):
         if hasattr(func, "_flow_group_operation_directives"):
             if self.name in func._flow_group_operation_directives:
-                raise ValueError(
+                raise FlowProjectDefinitionError(
                     f"Cannot set directives because directives already exist "
                     f"for {func} in group {self.name}"
```

Comment:
```suggestion
                    "Cannot set directives because directives already exist "
                    f"for operation '{func}' in group '{self.name}'."
```